src/color_laws.jl

```diff
@@ -372,9 +372,9 @@ function f99_invum(x::Real, Rv::Real)
 end
 
 # read and unpack tabulated data
-fpath = joinpath(dirname(pathof(DustExtinction)), "data/F19_tabulated.dat")
```

Comment:
Sorry, think my question was lost in the merge. Should we be worried that AppVeyor didn't seem to catch the forward slash I accidentally left behind?